src/collections/events/kubecon-china-2021/index.mdx

```diff
@@ -95,7 +95,12 @@ Speakers:
 
 Speakers:
 <ul>
-<li>All Service Mesh Performance project maintainers</li>
+<li><Link to="/community/members/aisuko-li">Sunku Ranganath</Link></li>
```

Comment:
A small mistake, Sunku's profile was linked to Aisuko, and the same mistake in Chinese version.

Reply:
Uh-oh. Thanks for pointing this out.